### ISSUE TITLE
SVG <mpath> sub-element of the <animateMotion> doesn't work in Playground

### ISSUE THREAD

**Issue by KotaruS** [mdn/content#28222]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/SVG/Element/mpath

### What specific section or headline is this issue about?

Example

### What information was incorrect, unhelpful, or incomplete?

The mpath element doesn't seem to work at all within MDN playground and thus in all examples. 
(I've checked that by editing HTML straight up and inserting the example it works, so browser support is not an issue :: tested on Microsoft Edge 115.0.1901.183 MacOS 13.4)
![image](https://github.com/mdn/content/assets/34488740/39ada3fe-5236-485e-bb83-392f1ece4da1)

### What did you expect to see?

Working example of a moving triangle along the SVG path.
![image](https://github.com/mdn/content/assets/34488740/630c8d73-da25-4c49-a1dc-9be85a04c15a)

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/svg/element/mpath`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/SVG/Element/mpath
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/svg/element/mpath/index.md
* Last commit: https://github.com/mdn/content/commit/b4f998244660723175f8e06b5d77f68cfb1d1f1a
* Document last modified: 2023-03-06T23:38:10.000Z

</details>

**Comment by sideshowbarker** [comment]
I can confirm that the SVG in https://developer.mozilla.org/en-US/docs/Web/SVG/Element/mpath#svg and that (as @KotaruS also noted) if you load that SVG outside the context of the Playground, the animation works as expected — but the animation does not work when loaded from within the MDN article and from the Playground.

I thought the cause could be the Content-Security-Policy that the SVG document is being served with — but I found that changing that doesn’t seem to make any difference. I thought that the cause might could also be the value of `sandbox` attribute on the iframe — but also there, I found that changing that doesn’t seem to make any difference.

I also checked the console messages in devtools in Firefox, Safari, and Chrome, but I found nothing getting logged there that offered any clues.
